e2e/testdata/fn-eval/output-to-stdout/.expected/exec.sh

```diff
@@ -16,4 +16,4 @@
 set -eo pipefail
 
 kpt fn source |\
```

Comment:
Are there e2e tests for output of source?

Reply:
Source is not the focus of this PR and not made any changes to that command.